src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -40,31 +40,25 @@ var (
 // the cloud, in command line flags, or in local customizations during
 // development.
 type Config struct {
-	configDir          string
-	Version            int
-	CloudConfig        string
-	CloudConfigCA      string
-	FrontedCloudConfig string
-	CPUProfile         string
-	MemProfile         string
-	UpdateServerURL    string
-	Client             *client.ClientConfig
-	ProxiedSites       *proxiedsites.Config // List of proxied site domains that get routed through Lantern rather than accessed directly
-	TrustedCAs         []*fronted.CA
-}
-
-// Fetcher is an interface for fetching config updates.
-type Fetcher interface {
-	pollForConfig(ycfg yamlconf.Config, sticky bool) (mutate func(yamlconf.Config) error, waitTime time.Duration, err error)
+	configDir       string
+	Version         int
+	CloudConfigCA   string
+	CPUProfile      string
+	MemProfile      string
+	UpdateServerURL string
+	Client          *client.ClientConfig
+	ProxiedSites    *proxiedsites.Config // List of proxied site domains that get routed through Lantern rather than accessed directly
+	TrustedCAs      []*fronted.CA
 }
 
 // StartPolling starts the process of polling for new configuration files.
-func StartPolling() {
+func (cfg *Config) StartPolling() {
```

Comment:
Why did you change polling to refer to a specific Config instance?  Polling is a process that continues forever, even though the underlying config instance changes over time.  Also, there's nothing in here that specifically refers to the cfg instance, so there's no benefit to it being an instance method.  Plus it refers to the global yamlconf.Manager.